client/components/chart/index.js

```diff
@@ -2,79 +2,66 @@
 /**
  * External dependencies
  */
-import { __ } from '@wordpress/i18n';
+import React from 'react';
 import classNames from 'classnames';
-import { Component, Fragment } from '@wordpress/element';
+import { isEqual } from 'lodash';
+import { Component } from '@wordpress/element';
+import PropTypes from 'prop-types';
 
 /**
  * Internal dependencies
  */
-import Card from 'components/card';
 import D3Chart from './charts';
-import dummyOrders from './test/fixtures/dummy';
 import Legend from './legend';
+import { gap, gaplarge } from 'stylesheets/abstracts/_variables.scss';
 
 const WIDE_BREAKPOINT = 1130;
```

Comment:
The wide breakpoint is actually 1100, this was changed from the initial designs recently.

Reply:
Thanks, @ryelle . Is there any benefit in getting breakpoint constants from a single source? We can import them from the SASS variables using `:export`, kinda like I did here - https://github.com/woocommerce/wc-admin/pull/270/files#diff-2f0350f29d44d2c8d79313778e03c3edR22.